src/images/map-legend.svg

```diff
@@ -1,27 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <svg width="40px" height="40px" viewBox="0 0 40 40" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-    <title>B2740935-CDFB-44E0-A95A-DF4D2EC0B091</title>
-    <defs>
-        <circle id="path-1" cx="20" cy="20" r="20"></circle>
-        <filter x="-21.2%" y="-16.2%" width="142.5%" height="142.5%" filterUnits="objectBoundingBox" id="filter-2">
-            <feOffset dx="0" dy="2" in="SourceAlpha" result="shadowOffsetOuter1"></feOffset>
-            <feGaussianBlur stdDeviation="2.5" in="shadowOffsetOuter1" result="shadowBlurOuter1"></feGaussianBlur>
-            <feComposite in="shadowBlurOuter1" in2="SourceAlpha" operator="out" result="shadowBlurOuter1"></feComposite>
-            <feColorMatrix values="0 0 0 0 0   0 0 0 0 0   0 0 0 0 0  0 0 0 0.2 0" type="matrix" in="shadowBlurOuter1"></feColorMatrix>
-        </filter>
-        <filter id="filter-3">
-            <feColorMatrix in="SourceGraphic" type="matrix" values="0 0 0 0 0.330000 0 0 0 0 0.330000 0 0 0 0 0.330000 0 0 0 1.000000 0"></feColorMatrix>
-        </filter>
-    </defs>
     <g id="FMC-Main" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
         <g id="Apps/HBI/map/kataster-no-legend-mobile" transform="translate(-303.000000, -299.000000)">
             <g id="basics/button/map/button-map-legend" transform="translate(303.000000, 299.000000)">
                 <g id="background-button-round">
-                    <use fill="black" fill-opacity="1" filter="url(#filter-2)" xlink:href="#path-1"></use>
+                    <use fill="black" fill-opacity="1" xlink:href="#path-1"></use>
                     <circle stroke="#545454" stroke-width="1" stroke-linejoin="square" fill="#FFFFFF" fill-rule="evenodd" cx="20" cy="20" r="19.5"></circle>
                 </g>
                 <g id="basics/icon/icon-legend" stroke-linecap="round" stroke-linejoin="round">
-                    <g filter="url(#filter-3)" id="Group">
+                    <g id="Group">
```

Comment:
Why have you removed this ? Now the colors between `location.svg` and this file differ.

Reply:
https://github.com/FixMyBerlin/fixmy.frontend/pull/960/commits/c9c2f68d642329a45b8fd417407d77558d027353

>The gray of the outline was lighter than the gray of the location icon below due to this svg filter stuff.

It looks good to me on https://deploy-preview-960--fixmyberlin-staging.netlify.app/planungen 
![image](https://user-images.githubusercontent.com/111561/151361317-a8277441-ae41-44d6-a715-a7918d2b86c5.png)

Which is better than it did with the filters present; it was too light then. Both SVG use the same color hex value now without filter, so that should be good, right?

Reply:
Mh… seeing the screenshots up close, the new icon is darker. But still, it should be easier to adjust without the filter, right?

Reply:
That's correct. But also the filter contained a Gaussian blur :)